src/pallavi/A12_ScannerProg3.java

```diff
@@ -0,0 +1,79 @@
+/* Assignment - 12 : 26th Dec'2021
+
+ 3. find the first non repeating character from given string. [program-3]
+   input : technocredits
+   output : h
+4. find the first repeating character from given string. [program-3]
+   input : shivangi
+   output : i
+5. write a method to return the reverse string. [program-3]
+ */
+
+package pallavi;
+
+import java.util.Scanner;
+
+public class A12_ScannerProg3 {
+
+	int findCountOfChar(String input, char c) {
+		int count = 0;
+
+		for (int i = 0; i < input.length(); i++) {
+			if (input.charAt(i) == c) {
+				count++;
+			}
+
+		}
+		return count;
+
+	}
+
+	void getNonRepetingChar(String input) {
+		int count = 0;
+		for (int index = 0; index < input.length(); index++) {
+			count = findCountOfChar(input, input.charAt(index));
+			if (count == 1) {
+				System.out.println("First non repeting char is:" + input.charAt(index));
+
+			}
+			break;
```

Comment:
this break should be inside if condition as when the count is only 1 that time you want to break the loop.